manager/src/main/java/io/streamzi/openshift/API.java

```diff
@@ -154,14 +153,14 @@ public void postFlow(String flowJson) {
     public String getGlobalProperties() {
         final Properties props = new Properties();
 
-        String bootstrapServers = EnvironmentResolver.get(ProcessorConstants.KAFKA_BOOTSTRAP_SERVERS);
+        final String bootstrapServers = EnvironmentResolver.get(ProcessorConstants.KAFKA_BOOTSTRAP_SERVERS);
```

Comment:
These are no longer used and can be removed. I've added a task to the spreadsheet to do this.

Reply:
I will assign that task to me :-)

Reply:
Assigned... :)

Reply:
ok, to keep things clean, that's a different PR